integrations/gitlab/gitlab_integration/events/hooks/push.py

```diff
@@ -49,3 +49,9 @@ async def _on_hook(self, body: dict[str, Any], gitlab_project: Project) -> None:
                 f"Updating project information after push hook for project {gitlab_project.path_with_namespace}"
             )
             await ocean.register_raw(ObjectKind.PROJECT, [gitlab_project.asdict()])
+        else:
+            logger.info(
+                "The event was not sent on a branch that matches the configured branch, "
+                "therefore the changes will not be imported to port. The event's ref: "
+                f"{ref}, the configured branch as ref: {generate_ref(branch)}"
+            )
```

Comment:
this is too spammy and long, I wouldn't want to add this log as I suspect most of the cases there are more commits to other branch ( while developing ) than the main branch, so it means the user will this message too much

Reply:
you are correct, though when you get an event like that, you will log all of the info logs of the handling anyway, and wouldn't be able to realize if there is a bug or the branch is irrelevant.